src/bin/potr/opts.rs

```diff
@@ -52,6 +52,14 @@ pub struct Opts {
     /// Process code blocks (starts with ```). By default, code blocks are skipped.
     #[clap(long, visible_alias = "pc")]
     pub process_code_blocks: bool,
+    
+    /// Skip $$ prefixed message(skip formula) (starts with $$). By default, $$ blocks are skipped.
+    #[clap(long, visible_alias = "sf")]
+    pub skip_formula_blocks: bool,
```

Comment:
I believe by default, we don’t want to process these blocks, so in the command line, it is better to use process-* instead of skip-*, otherwise we have to specify it every time.

Reply:
you're right!